source/triggers/scheduled-triggers.txt

```diff
@@ -113,11 +116,34 @@ Scheduled Triggers have the following configuration options:
        
        For scheduled Triggers, this value should be set to ``SCHEDULED``.
 
-   * - | :guilabel:`Trigger Name`
+   * - | :guilabel:`Name`
        | ``name: <string>``
 
      - Required. The name of the Trigger.
 
+   * - | :guilabel:`Enabled`
+       | ``disabled: <boolean>``
+
+     - Enabled by default. Used to enable trigger. 
```

Comment:
```suggestion
     - Enabled by default. Used to enable the Trigger. 
```
i'm a little confused by how we're using capital "T" Trigger vs. trigger -- seems inconsistent